.travis.yml

```diff
@@ -33,6 +33,8 @@ jobs:
       if: (commit_message !~ /^.*(only:).*$/) OR (commit_message =~ concat(^.*,env(DIR),.*$))
     - env: DIR=nyc_taxi
       if: (commit_message !~ /^.*(only:).*$/) OR (commit_message =~ concat(^.*,env(DIR),.*$))
+    - env: DIR=opensky
```

Comment:
It sure would be nice if .travis.yml didn't have to hard-code the name of each project, so that projects can be added and deleted without editing this shared file.